### PR TITLE
Bootstrap the CSHZ Sim

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: circumstellar-habitable-zone-simulator/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  "homepage": "https://github.com/ccnmtl/astro-simulations#readme",
+  "dependencies": {
+    "@babel/cli": "^7.5.5",
+    "eslint-plugin-security": "^1.4.0",
+    "pixi.js": "~5.3.0",
+    "react": "~17.0.1",
+    "react-dom": "~17.0.1",
+    "react-range-step-input": "^1.2.1",
+    "three": "~0.129.0"
```
Comment (nikolas) [inline circumstellar-habitable-zone-simulator/package.json:34]: You can probable take out three.js from the dependencies here.

File: circumstellar-habitable-zone-simulator/package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "author": "Nik Nyby",
+  "license": "GPL-3.0+",
+  "bugs": {
+    "url": "https://github.com/ccnmtl/astro-simulations/issues"
+  },
+  "homepage": "https://github.com/ccnmtl/astro-simulations#readme",
+  "dependencies": {
+    "@babel/cli": "^7.5.5",
+    "eslint-plugin-security": "^1.4.0",
+    "pixi.js": "~5.3.0",
```
Comment (nikolas) [inline circumstellar-habitable-zone-simulator/package.json:30]: I would use the latest version of pixi to start: 6.0.4. There's a small bug with SVG loading in 6.0.4, but it's easy to work around, and will be fixed soon anyways: https://github.com/pixijs/pixijs/pull/7478

File: circumstellar-habitable-zone-simulator/package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "eslint": "eslint src/*.js src/*.jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ccnmtl/astro-simulations.git"
+  },
+  "keywords": [
+    "astronomy"
+  ],
+  "author": "Nik Nyby",
```
Comment (nikolas) [inline circumstellar-habitable-zone-simulator/package.json:21]: Update the author to yourself!

Comment (nbuonin) [reply circumstellar-habitable-zone-simulator/package.json]: Hah, ok, I'll stop being shy!

File: circumstellar-habitable-zone-simulator/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    mode: process.env.WEBPACK_SERVE ? 'development' : 'production',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'bundle.js',
+        publicPath: '/dist/',
+    },
+    resolve: {
+        extensions: ['*', '.js', '.jsx'],
+        alias: {
+            'three/CopyShader': path.join(__dirname, 'node_modules/three/examples/js/shaders/CopyShader.js')
```
Comment (nikolas) [inline circumstellar-habitable-zone-simulator/webpack.config.js:15]: All this three.js config stuff in the webpack config can be removed.

**Pull request merged**